tempest2/src/main/kotlin/app/cash/tempest2/internal/DynamoDbLogicalDb.kt

```diff
@@ -346,7 +346,9 @@ internal class DynamoDbLogicalDb(
     // parse the potentially concurrency related TransactionCancelledExceptions
     // https://docs.aws.amazon.com/AWSJavaSDK/latest/javadoc/com/amazonaws/services/dynamodbv2/model/TransactionCanceledException.html
     throw TransactionCanceledException.builder()
-      .message("Write transaction failed: ${writeSet.describeOperations()}")
+      .message(
+        "Write transaction failed: ${writeSet.describeOperations()}. Aws error message: ${e.message}"
```

Comment:
It might be nice to put a newline before `Aws error message`, `describeOperations()` can contain them so it will have an awkward placement otherwise

Reply:
I added a newline 🙏🏼